local/convergence.go

```diff
@@ -240,8 +240,12 @@ func (s *composeService) runContainer(ctx context.Context, project *types.Projec
 		return err
 	}
 	id := created.ID
-	for net := range service.Networks {
-		name := fmt.Sprintf("%s_%s", project.Name, net)
+	for netName := range service.Networks {
+		network := project.Networks[netName]
+		name := fmt.Sprintf("%s_%s", project.Name, netName)
+		if network.External.External {
+			name = network.Name
```

Comment:
IIRC, network name should be the one set in compose file (when set) even if this is a managed one (external=false). Would need to double check with docker-compose

Reply:
Indeed, tested with docker-compose, and if network name is specified in yaml, it's used as is, no prefix

Reply:
This network name setting needed to be fixed earlier, when creating/checking networks, in compose.go. Here we can just use the network.name that is set in the project.